lib/core/src/lib/avatar/avatar.component.spec.ts

```diff
@@ -86,4 +86,16 @@ describe('AvatarComponent', () => {
         const avatarElement: HTMLElement = fixture.nativeElement.querySelector('.adf-avatar__image');
         expect(avatarElement.getAttribute('title')).toBe('User Tooltip');
     });
+
+    it('should call onImageError when the image fails to load', () => {
+        spyOn(component, 'onImageError').and.callThrough();
+        component.src = 'path/to/image.jpg';
+        fixture.detectChanges();
+
+        const avatarElement: HTMLElement = fixture.nativeElement.querySelector('.adf-avatar__image');
```

Comment:
I see in this file there is many usages of 
const avatarElement: HTMLElement = fixture.nativeElement.querySelector('.adf-avatar__image');

can we get this element in beforeEach OR can we create function to get this element to avoid repeating looking for it?